pxtsim/simdriver.ts

```diff
@@ -320,8 +321,8 @@ namespace pxsim {
                 if (parentWindow) {
                     // if message comes from parent already, don't echo
                     if (source !== parentWindow) {
-                        const parentOrigin = this.options.parentOrigin || window.location.origin
-                        parentWindow.postMessage(msg, parentOrigin);
+                        // posting sim messages to parent frame; no origin restriction.
```

Comment:
also this one's worth noting -- we hit this when we send messages from sim to parent frames, but we really don't care about blocking those as there is nothing 'secret' in them. (for our case, clearing this up gets rid of about 3k errors / minute & is required to pass around messages nicely on localhost.)